modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple data centers and clusters, thus helping to avoid a single-point-of-failure.
+
+:FeatureName: vSphere topology Awareness
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+.Procedure
+To deploy {product-title} for vSphere storage on different zones and regions:
+
+. In the VMware vCenter vSphere client GUI:
+.. Create and assign zone and region type tags as needed to the hosts on the desired clusters.
+.. Create and apply "cat-zone-local-datastore" tags to the desired datastores.
+
++
+For more information about vSphere tags, see the WMware vSphere documentation.
```

Comment:
Should we link to the VMWare vSphere documentation? link:URL[WMware vSphere documentation]? 

Edit: I see it's included below.